### ISSUE TITLE
tooltip is not working / duplicated when using tooltipHTML

### ISSUE THREAD

**Issue by puemos** [amcharts/amcharts4#28]
1. You have to set tooltipText so that the tooltip will work [reference](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/.internal/core/Sprite.js#L6561)

```js
if (tooltip.text != undefined && tooltip.text != "") {
                    //@todo: think of how to solve this better
                    if (tooltip && !tooltip.parent) {
                        tooltip.parent = this.tooltipContainer;
                    }
                    // Reveal tooltip
                    tooltip.show();
                    return true;
}
```

2. Every time you move to a new column the tooltip is duplicated [example](https://codepen.io/puemos/pen/QrZdJy)

![may-16-2018 17-44-51](https://user-images.githubusercontent.com/13174025/40127983-dceea430-5930-11e8-8a25-967f13b20b6e.gif)

**Comment by martynasma** [comment]
Looks like the issue is even deeper. `tooltipHTML` is ignored altogether if `tooltipText` is not set, which does not make a lot of sense.

Will be fixing it.

**Comment by martynasma** [comment]
OK, got that little bugger tackled. Will be rolled into next release within a few days.

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.25](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.25)
